Lib/ftplib.py

```diff
@@ -105,12 +103,13 @@ class FTP:
     passiveserver = 1
     encoding = "latin-1"
 
-    # Initialization method (called by class instantiation).
-    # Initialize host to localhost, port to standard ftp port
-    # Optional arguments are host (for connect()),
-    # and user, passwd, acct (for login())
     def __init__(self, host='', user='', passwd='', acct='',
                  timeout=_GLOBAL_DEFAULT_TIMEOUT, source_address=None):
+        """Initialization method (called by class instantiation).
+        Initialize host to localhost, port to standard ftp port
+        Optional arguments are host (for connect()),
+        and user, passwd, acct (for login())
+        """
```

Comment:
This change is good, ``FTP.__init__`` is a public method and didn't have a docstring previously.